gradio/components.py

```diff
@@ -1929,6 +1931,14 @@ def postprocess(self, y: str) -> str:
         if y is None:
             return None
         returned_format = y.split(".")[-1].lower()
```

Comment:
Unreleated to this pr but I feel like we should use a NamedTemporaryFile here like we do for Image?

Reply:
#1967 takes care of this